ws-nextjs-app/pages/[service]/new_live/[id]/LivePageLayout.tsx

```diff
@@ -5,16 +5,16 @@ import { jsx } from '@emotion/react';
 import Heading from '#app/components/Heading';
 import { ServiceContext } from '#contexts/ServiceContext';
 import nodeLogger from '#lib/logger.node';
-import LegacyText from '#app/legacy/containers/Text';
-import Blocks from '#app/legacy/containers/Blocks';
 import Pagination from '#app/components/Pagination';
 import MetadataContainer from '../../../../../src/app/components/Metadata';
 import LinkedDataContainer from '../../../../../src/app/components/LinkedData';
 import Stream from './Stream';
 import Header from './Header';
+import KeyPoints from './KeyPoints';
 
 import styles from './styles';
 import { StreamResponse, Page } from './Post/types';
+import { SummaryContent } from './KeyPoints/types';
```

Comment:
Is the name difference of Summary and KeyPoints going to get confusing? Do we want to just stick with one name throughout?

Reply:
Ahh good point! So I renamed it to Key Points since Product are calling it 'Key Points'. But in the Live Experience response it's 'Summary Points' (and this is what we get passed through from the BFF).

I will update it to make it more consistant, but I'm not sure if I've made it more confusing by calling the component 'Key Points' instead of 'Summary'.